meta/sai_meta.cpp

```diff
@@ -6757,3 +6757,283 @@ sai_status_t meta_sai_flush_fdb_entries(
 
     return flush_fdb_entries(switch_id, attr_count, attr_list);
 }
+
+// NAT
+
+sai_status_t meta_sai_validate_nat_entry(
+        _In_ const sai_nat_entry_t* nat_entry,
+        _In_ bool create)
+{
+    SWSS_LOG_ENTER();
+
+    if (nat_entry == NULL)
+    {
+        SWSS_LOG_ERROR("nat_entry pointer is NULL");
+
```

Comment:
extra newline. Same for many lines

Reply:
Inline with newlines added in other similar functions like meta_sai_create_fdb_entry and meta_sai_validate_fdb_entry. Let me know if the newlines are not needed.